pkg/ipamd/datastore/data_store.go

```diff
@@ -34,6 +34,9 @@ const (
 	// in addressCoolingPeriod
 	addressCoolingPeriod = 30 * time.Second
 
+	// addressDelayedReleasePeriod is used to release unused IP after a period of inactivity.
+	addressDelayedReleasePeriod = 10 * time.Minute
```

Comment:
Still reviewing it, but on a high level this should be configurable.

Reply:
Sure, I can make it configurable.
afair there was a discussion in related issue that we don't want to introduce new env variables. Is it ok to expose it via env var with some safe default value?